themes/custom/droopler_theme/scss/components/_main-menu.scss

```diff
@@ -457,7 +457,6 @@ header.header > nav.navbar-wrapper {
     nav.main-navbar {
       position: absolute;
       top: 100%;
-      right: 0;
```

Comment:
wait, i'm not sure about removing this one - it was added for a reasen.. i'll check it

Reply:
correct, this `right` (or `left`) setting here seems required, without it we have horizontal scrollbar:
![image](https://user-images.githubusercontent.com/37533838/71666738-8fc02280-2d62-11ea-86f1-fbbcd780613b.png)